core/filestorage/filesync/filesync.go

```diff
@@ -110,7 +110,7 @@ func (f *fileSync) Run(ctx context.Context) (err error) {
 	}
 
 	// TODO multi-spaces: init for each space: GO-1681
-	{
+	go func() {
 		spaceID := f.spaceService.AccountId()
 		_, err = f.SpaceStat(ctx, spaceID)
```

Comment:
race detector can dislike it :) rename err or a better way to make a separate func

Reply:
thanks, fixed